### PR TITLE
RavenDB-18438 PostgreSQL View - don't call server when user doesn't h…

### PR DESCRIPTION
**Pull request by ml054**
…ave valid license attributes.

### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-18438 

### Additional description

we check license attributes and if user doesn't have power bi nor PostgreSql we display small warning. If he tries to save credentials we show full license limit exception with ability to upgrade license.

### Type of change

- Optimization

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### CODE REVIEW

File: src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html [hunk 1]
```diff
@@ -39,6 +39,14 @@ <h4 class="text-elipsis margin-left">
                         <span>.</span>
                     </small>
                 </div>
+                <div class="margin-top padding padding-xs text-warning bg-warning" data-bind="visible: needsLicenseUpgrade">
+                    <small>
+                        <i class="icon-warning"></i>
+                        <span>
+                            You cannot use this feature because your license doesn't allow neither PostgreSQL integration feature nor Power BI
```
Comment (Danielle9897) [inline src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html:46]: double negative.. so:

=>
* In order to use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.

or 

* This feature is not available as your license allows neither PostgreSQL integration nor Power BI features

or

* This feature is not available as your license doesn't include either the PostgreSQL integration feature or the Power BI feature

Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html:46]: let's go with: In order to use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html:46]: I would go with 1'st bullet from above
We can make it shorter as follows:

`To use this feature, your license must include either of the following features: PostgreSQL integration or Power BI.`

Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/database/settings/integrations.html:46]: https://github.com/ravendb/ravendb/pull/14163

**Pull request merged**